src/client/dfs/dfs.c

```diff
@@ -65,7 +65,9 @@
 #define CTIME_IDX	(MTIME_IDX + sizeof(time_t))
 #define CSIZE_IDX	(CTIME_IDX + sizeof(time_t))
 #define OCLASS_IDX	(CSIZE_IDX + sizeof(daos_size_t))
-#define SYML_IDX	(OCLASS_IDX + sizeof(daos_oclass_id_t))
+#define UID_IDX		(OCLASS_IDX + sizeof(uid_t))
+#define GID_IDX		(UID_IDX + sizeof(gid_t))
+#define SYML_IDX	(GID_IDX + sizeof(daos_oclass_id_t))
```

Comment:
Should this not be?
```suggestion
#define UID_IDX		(OCLASS_IDX + sizeof(daos_oclass_id_t))
#define GID_IDX		(UID_IDX + sizeof(uid_t))
#define SYML_IDX	(GID_IDX + sizeof(daos_oclass_id_t))
```

Reply:
yes not sure how i missed that.. and im concerned that it's working as is :-/